### ISSUE TITLE
Oops There was an error processing your request with ID: ...

### ISSUE THREAD

**Issue by berndhahnebach** [AECgeeks/ifc-pipeline#4]
find attached an very simple ifc including a IfcSpace. This can somehow not be processed from the viewer. If the IfcSpace is changed to an IfcSlab by an editor it will be processed. Might be something is wrong on parsing IfcSpace?!

cheers bernd

[space.ifc.txt](https://github.com/AECgeeks/ifc-pipeline/files/5231630/space.ifc.txt)

**Comment by aothms** [comment]
yes, this is just a bit silly, we depend no the status code from IfcConvert and when it doesn't convert a single element successfully it will return a non-zero exit code and IfcSpace is excluded by default by the entity filter. Both should be fixed probably, I'll keep this issue open as a reminder. Thanks.

**Comment by berndhahnebach** [comment]
I must admit in most cases I do not want to import IfcSpaces. Thus ignoreing them is fine. But lately I have been done some projects in the regard of IfcSpace. As a result I get Ifcs which have IfcSpace and really need them. For these workflow and only for these workflow it would be cool to parse IfcSpace as well. Eventuale some preference in this regard would be great.
